### DISCUSSION TITLE
Multi-Tenant Admin Panel

### DISCUSSION THREAD

**Discussion by kaloyan-manev-dl** [payloadcms/payload#7619] [Feature Requests & Ideas]
Hello everyone!

First, I want to say that Payload is truly amazing!

I've been reading the topic about multi-tenancy and I was wondering if it is possible to include a tenant dropdown in the login screen? This would be a nice feature to have. It would allow an admin to use a single login panel to enter all the available tenants. 

A possible solution is to have a custom admin login component. What do you think?

**Comment by roybarber** [comment]
@kaloyan-manev-dl Take a look at the example in the v3 beta branch:

https://github.com/payloadcms/payload/tree/beta/examples/multi-tenant-single-domain/src/components/TenantSelector

However when you login as SuperAdmin in the above example, you can see all the tenants, their users, pages etc
